Profile

```diff
@@ -0,0 +1 @@
+web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
```

Comment:
are you using this?

Reply:
because it was a config suggestion. 
create a profile file and send this command

Reply:
what change if we remove or add it?

Reply:
the server already started on the puma, but as was suggested I wanted to try